tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1240,4 +1245,198 @@ public function checkUserInformation(array $expectedValue, array  $actualValue):
 			}
 		}
 	}
+
+	/**
+	 * @When user :byUser tries to get information of user :user using Graph API
```

Comment:
```suggestion
	 * @When user :byUser tries to get information of user :user using Graph API
	  * @When user :byUser gets information of user :user using Graph API
```
We can do multiple steps with a single implementation. Only, if you use the suggested version in the feature file.